moe.karaokes.mugen.app.yml

```diff
@@ -0,0 +1,93 @@
+app-id: moe.karaokes.mugen.app
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
```

Comment:
```suggestion
runtime-version: '22.08'
```

Reply:
Regarding the 22.08 runtime version instead of 21.08, we tested with 22 but it threw a weird node exception we didn't know how to fix, We can look into this further in later releases, unless 22 is mandatory?

Reply:
This should be working now, we upgraded to 22.08 and it builds/runs on our end.